src/modules/tracker.js

```diff
@@ -203,6 +259,92 @@ class Tracker {
     return new Error('parameters are required of type object');
   }
 
+  /**
+   * Send autocomplete select event to API
+   * @private
+   * @function trackAutocompleteSelectV2
+   * @param {string} item_name - Name of selected autocomplete item
+   * @param {object} parameters - Additional parameters to be sent with request
+   * @param {string} parameters.user_input - The current autocomplete search query
+   * @param {string} [parameters.section] - Section the selected item resides within
+   * @param {string} [parameters.tr] - Trigger used to select the item (click, etc.)
+   * @param {string} [parameters.item_id] - Item id of the selected item
+   * @param {string} [parameters.variation_id] - Variation id of the selected item
+   * @param {string} [parameters.group_id] - Group identifier of selected item
+   * @param {object} [networkParameters] - Parameters relevant to the network request
+   * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
+   * @returns {(true|Error)}
+   * @description User selected (clicked, or navigated to via keyboard) a result that appeared within autocomplete
+   * @example
+   * constructorio.tracker.trackAutocompleteSelectV2(
+   *     'T-Shirt',
+   *      {
+   *          user_input: 'Shirt',
+   *          section: 'Products',
+   *          tr: 'click',
+   *          group_id: '88JU230',
+   *          item_id: '12345',
+   *          variation_id: '12345-A',
+   *      },
+   * );
+   */
+  trackAutocompleteSelectV2(item_name, parameters, networkParameters = {}) {
+    // Ensure term is provided (required)
+    if (item_name && typeof item_name === 'string') {
+      // Ensure parameters are provided (required)
+      if (parameters && typeof parameters === 'object' && !Array.isArray(parameters)) {
+        const baseUrl = `${this.behavioralV2Url}autocomplete_select?`;
+        const {
+          original_query,
+          user_input = original_query,
+          original_section,
+          section = original_section,
+          tr,
+          group_id,
+          item_id,
+          variation_id,
+        } = parameters;
+        const queryParams = {};
+        const bodyParams = {
```

Comment:
I think we're missing `section` in `bodyParams`

Reply:
Same as previous